danger_code_ws/1.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <cstdio>
+
+int main() {
```

Comment:
Consider wrapping the `main` function in a namespace to avoid potential naming conflicts.  This is especially important in larger projects. For example:
```c++
namespace my_program {
int main() {
  // ... your code ...
}
} // namespace my_program
```